src/components/common/MwInput/MwInput.scss

```diff
@@ -1,4 +1,5 @@
 @import '../../../styles/colors.scss';
+@import '../../Header/AddEditDialog/AddEditDialog.scss';
```

Comment:
Imported my mistake, I guess

Reply:
no, it was done intentionally AddEditDialog.scss has common things for three components, I know that it is better to move it to some common styles sheets, but here I decides to simplify